app/renderer/actions/Session.js

```diff
@@ -719,9 +720,33 @@ export function setVisibleProviders () {
  */
 function includeSafariInWebviews (caps) {
   const {browserName = ''} = caps;
+  const safariCapabilities = {
+    includeSafariInWebviews: true,
+  };
+
+  return {
+    ...caps,
+    ...(browserName.toLowerCase() === 'safari' ? safariCapabilities : {}),
+  };
+}
+
+/**
+ * Check if Chrome is started, if so set the ChromeDriver to w3c:false because
+ * all internal calls are still JSONWP calls
+ *
+ * @param {object} caps
+ */
+function setChromeDriverToJsonwp (caps) {
+  const {browserName = ''} = caps;
+  const chromeCapabilities = {
+    nativeWebScreenshot: true,
+    chromeOptions: {
+      'w3c': false,
```

Comment:
ah, yeah, https://github.com/admc/wd AD uses as WebDriver client is still JSONWP

Reply:
Yep, it broke a lot for my, then I figured out this was the best way